src/components/ContentLoader.tsx

```diff
@@ -32,7 +38,7 @@ export const ContentLoader = ({ contentId }: ContentLoaderProps): JSX.Element =>
       }
 
       const data = await response.json() as ContentResponse
-      const htmlContent = data.content[0].html
+      const htmlContent = getVariant(data.content)
```

Comment:
See related comment in `utils.ts`, but it'd be nice to set an error if we somehow don't find an expected variant in the context (either `main` or the one defined in `variant-mapping.json`)